dev-util/spirv-tools/spirv-tools-1.3.239.ebuild

```diff
@@ -34,10 +35,9 @@ BDEPEND="${PYTHON_DEPS}
 
 multilib_src_configure() {
 	local mycmakeargs=(
-		"-DSPIRV-Headers_SOURCE_DIR=${ESYSROOT}/usr/"
-		"-DSPIRV_WERROR=OFF"
-		"-DSPIRV_TOOLS_BUILD_STATIC=OFF"
-		"-DBUILD_SHARED_LIBS=ON"
+		-DSPIRV-Headers_SOURCE_DIR="${ESYSROOT}"/usr/
+		-DSPIRV_WERROR=OFF
+		-DSPIRV_TOOLS_BUILD_STATIC=OFF
```

Comment:
we're loosing `-DBUILD_SHARED_LIBS=ON`, is that intentional?

Reply:
It is set by default in `cmake.eclass`, see also commit message, see also `gentoo_common_config.cmake` or `CMakeCache.txt`.

Reply:
ah sorry i didnt look at the commits